aws_xray_sdk/core/patcher.py

```diff
@@ -12,6 +12,7 @@
 log = logging.getLogger(__name__)
 
 SUPPORTED_MODULES = (
+    'aiobotocore',
```

Comment:
Can you add 'aiobotocore' as an item in the NO_DOUBLE_PATCH tuple as well?

Reply:
Done!